scripts/utils/integration-tests

```diff
@@ -47,10 +47,14 @@ TESTS_FOLDER="tests"
 
 PLUGINS_PATH=${PLUGINS_PATH:-''}
 
-while getopts 's:hnr:?' flag; do
+# if set, runs all tests at once
+RUN_ALL_TESTS=
+
+while getopts 's:ahnr:?' flag; do
   case "${flag}" in
     s) SEARCH=${OPTARG:-'*'} ;;
     h) CYPRESS_HEADED="true" ;;
+    a) RUN_ALL_TESTS="true" ;;
```

Comment:
Let's document this in the README please :)

Reply:
will do once I know how much improvement it brings, if its less than 10min, I'll close this PR anyway 👍